ports/yolkheroes/yolkheroes/yolkheroes.gptk

```diff
@@ -0,0 +1,16 @@
+up = up
+down = down
+left = left
+right = right
+
+a = z
+b = x
+x = tab
```

Comment:
In the readme both A and x is Z.
which one is correct?

Reply:
I must have swapped this for testing, will fix